docs/RestApis.md

```diff
@@ -155,6 +155,49 @@ Filter object is defines as below:
 
 **Content** : `Error message`
 
+## Parsed Query
+
+Adding a `parsed = true` parameter in the request body, the query response will be enriched with the destructured query.
```

Comment:
Maybe it would be possible to find a more specific word instead of destructured. `Analyzed` query ? `Tokenized` query ?

Reply:
we should change the PR title accordingly